addons/mail/static/src/models/thread_view/thread_view.js

```diff
@@ -359,6 +363,12 @@ function factory(dependencies) {
         deviceIsMobile: attr({
             related: 'device.isMobile',
         }),
+        /**
+         * Determines whether the input needs to have focus.
+         */
+        doFocusOnInput: attr({
+            default: false,
+        }),
```

Comment:
need "rename" on the name of the field, "input" by itself is not clear enough as to what should be focused